### ISSUE TITLE
A11y: low contrast orange

### ISSUE THREAD

**Issue by patricoferris** [ocaml/ocaml.org#201]
Through out the site we use an orange background with white text (e.g. for buttons) or the inverse (e.g. headings). As with the V2 version of the site (see https://github.com/ocaml/ocaml.org/issues/1199), this has a low contrast and fails web accessibility standards tests. 

More concretely the "Get Started" button on the top right has a background colour of `#ee6a1a` with white text. Using something like the [WebAIM contrast checker it scores 3.12](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=EE6A1A.) (ideally it would be greater than or equal to 7 or at least 4.5). Using the tool this gives two colours:

 - Contrast ratio 4.5: `#C2540F` (see [test](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=C2540F))
 - Contrast ratio 7: `#943F0A` (see [test](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=943F0A))

Which are heading towards brown more than orange. Any thoughts on a design to improve this (note this often applies to the links in text too which are orange on a pale background)?

**Comment by sabine** [comment]
Resolved in the meantime.
